### PR TITLE
 Updated zulip to discourse #625

### PR DESCRIPTION
**Pull request by aliya-rahmani**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
 .. |ReadTheDocs| image:: https://readthedocs.org/projects/cpython-devguide/badge/
    :target: https://devguide.python.org
    :alt: Documentation Status
 
-.. |Zulip| image:: https://img.shields.io/badge/zulip-join_chat-brightgreen.svg
-   :alt: Python Zulip chat
-   :target: https://python.zulipchat.com
+.. |Discourse| image:: https://img.shields.io/badge/discourse-join_chat-brightgreen.svg
+   :alt: Python Discourse chat
+   :target: https://pypi.org/project/pydiscourse/
```
Comment (methane) [inline README.rst:12]: Do we really need pydicourse on PyPI??

Comment (hugovk) [reply README.rst:12]: Link to here:
```suggestion
   :target: https://discuss.python.org/
```

**Comment by hugovk** [comment]
Also please can you change the PR to target `master` rather than `Mariatta-patch-1`?

**Comment by aliya-rahmani** [comment]
@hugovk I've made the changes in #630

**Pull request closed without merging**